runtime/common/metering.go

```diff
@@ -147,6 +147,17 @@ var (
 	PublicAccountContractsStringMemoryUsage = NewRawStringMemoryUsage(len("PublicAccount.Contracts()"))
 	AuthAccountKeysStringMemoryUsage        = NewRawStringMemoryUsage(len("AuthAccount.Keys()"))
 	PublicAccountKeysStringMemoryUsage      = NewRawStringMemoryUsage(len("PublicAccount.Keys()"))
+	CapabilityValueStringMemoryUsage       = NewRawStringMemoryUsage(len("Capability<>(address: , path: )"))
+	LinkValueStringMemoryUsage             = NewRawStringMemoryUsage(len("Link<>()"))
+
+	// Static types string representations
+
+	VariableSizedStaticTypeMemoryUsage = NewRawStringMemoryUsage(2)  // []
+	DictionaryStaticTypeMemoryUsage    = NewRawStringMemoryUsage(4)  // {: }
+	OptionalStaticTypeMemoryUsage      = NewRawStringMemoryUsage(1)  // ?
+	AuthReferenceStaticTypeMemoryUsage = NewRawStringMemoryUsage(5)  // auth&
+	ReferenceStaticTypeMemoryUsage     = NewRawStringMemoryUsage(1)  // &
+	CapabilityStaticTypeMemoryUsage    = NewRawStringMemoryUsage(12) // Capability<>
```

Comment:
These names are kind of misleading; can these be changed to reflect the fact that they are strings and not types themselves? 
```suggestion
	VariableSizedStaticTypeStringMemoryUsage = NewRawStringMemoryUsage(2)  // []
	DictionaryStaticTypeStringMemoryUsage    = NewRawStringMemoryUsage(4)  // {: }
	OptionalStaticTypeStringMemoryUsage      = NewRawStringMemoryUsage(1)  // ?
	AuthReferenceStaticTypeStringMemoryUsage = NewRawStringMemoryUsage(5)  // auth&
	ReferenceStaticTypeStringMemoryUsage     = NewRawStringMemoryUsage(1)  // &
	CapabilityStaticTypeStringMemoryUsage    = NewRawStringMemoryUsage(12) // Capability<>
```

Reply:
That makes sense! Added with f444896a1f5b9f0aaba562287d4c021c4e63df6f